bosh-setup/README.md

```diff
@@ -4,11 +4,13 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
```

Comment:
qingfuwang has left our team.
Currently I'm maintaining the template.
There are three options:
1. githubUsername can be a list. I can put qingfuwang and me into it.
2. Replace qingfuwang with my own githubusername
3. Add a contributor tag which includes a list of contributor to the template.

Currently, should I choose the second option?